libs/components/router/src/lib/modules/link/link-external.directive.spec.ts

```diff
@@ -42,9 +42,11 @@ describe('SkyAppLinkExternal Directive', () => {
 
   function setup(
     params: any,
-    windowName: string,
+    windowName: string | undefined,
```

Comment:
```
windowName?: string,
```
You might need to put this parameter at the end of the params list.

Reply:
Fixed. Though I did have to reorder all of the calls to `setup` for this one as you can't have optional params before required ones.